sdk/golang/ccnp/quote/proto/quote-server.pb.go

```diff
@@ -0,0 +1,428 @@
+// Code generated by protoc-gen-go. DO NOT EDIT.
```

Comment:
I'am thinking about why not move such `protoc-gen-go` and `protoc-gen-go-grpc` work into Makefile, so that such xxxx.pb.go and xxx_grpc.pb.go can be out of git scope and be generated in build time. 
Maybe not in this PR. We can talk about this later.